mvtuong/drinking
# sync game state

Includes the functionality to sync the game state between all users. Also retrieves the last active state when a new user navigates to the browser.

To test:

Open multiple tabs and add a new user on each tab. The state should be synced between browsers. At the moment I am only syncing users. Not any other data. We can do that in a separate PR.
